README.md

```diff
@@ -643,6 +643,24 @@ can be used to opt out of hoisting for certain dependencies.
 $ lerna bootstrap --hoist --nohoist=babel-*
 ```
 
+#### --npm-client [client]
+
+Install external dependencies using `[client] install`.  Must be an executable
+that knows how to install npm dependencies.
```

Comment:
Because that's the only command it's used for. 😁 

Maybe better to say "Must be a compatible npm registry client"?